channeldb/db.go

```diff
@@ -353,6 +354,28 @@ func (d *DB) FetchAllOpenChannels() ([]*OpenChannel, error) {
 	return fetchChannels(d, false, false)
 }
 
+// PendingChannelsCount will return the number of channels that have completed the process of
+// generating and broadcasting funding transactions, but whose funding
+// transactions have yet to be confirmed on the blockchain.
+func (d *DB) PendingChannelsCount(chainHash chainhash.Hash) (uint32, error) {
```

Comment:
PendingChannelsCount is used by the rpcserver (getinfo)  and channel_test.go.

It was not created for the PR and existed before.  I modified it to support multi-chain

Reply:
OK you're right, I can find it in the current codebase. It's just odd that the git diff makes it look like these lines are being added without anything similar removed.

Reply:
Actually, I'm wrong... I created it. The code used to check the number of pending channels with the len() function. under multi-chain, I can't used len() any more since I need to have the number of pending channel per chain. This is why I created the function.